deploy.sh

```diff
@@ -300,22 +320,24 @@ if [ -z "$DOCKER_ORGANISATION" ] ; then
   sed -e "s/<docker-id>/$DOCKER_USERNAME/" \
   -e "s/<prefix>/$DOCKER_IMAGE_PREFIX/" \
   -e "s/<jupyterhub-ip>/$JUPYTERHUB_IP/" \
-  ./config-template.yaml > ./config.yaml
+  ${DIR}/config-template.yaml > ${DIR}/config.yaml
 else
   sed -e "s/<docker-id>/$DOCKER_ORGANISATION/" \
   -e "s/<prefix>/$DOCKER_IMAGE_PREFIX/" \
   -e "s/<jupyterhub-ip>/$JUPYTERHUB_IP/" \
-  ./config-template.yaml > ./config.yaml
+  ${DIR}/config-template.yaml > ${DIR}/config.yaml
 fi
 
 echo "--> Updating Helm chart"
 helm upgrade $HELM_BINDERHUB_NAME jupyterhub/binderhub \
 --version=$BINDERHUB_VERSION \
--f ./secret.yaml \
--f ./config.yaml | tee helm-upgrade.log
+-f ${DIR}/secret.yaml \
+-f ${DIR}/config.yaml | tee helm-upgrade.log
 
 # Print Binder IP address
+echo "--> Retrieving Binder IP"
 BINDER_IP=`kubectl --namespace=$HELM_BINDERHUB_NAME get svc binder | awk '{ print $4}' | tail -n 1`
+echo "Binder IP: ${BINDER_IP}"
```

Comment:
```suggestion
echo "Binder IP: ${BINDER_IP}" | tee binder-ip.log
```
In the unlikely event it succeeds first time, ensure the miracle makes it into a log file :-)

Reply:
We do this on this line :) https://github.com/alan-turing-institute/binderhub-deploy/blob/90f7b62e6787667e3f2d80ee4d4f47e97bde8002/deploy.sh#L346

Reply:
However, this might not make it into the logs if the while loop is never triggered. So probably best to include this as a failsafe.